src/bitbucket/BitbucketAPI.ts

```diff
@@ -223,6 +223,28 @@ export class BitbucketAPI {
       }));
   };
 
+  getPullRequestPriority = async (commit: string): Promise<BB.PRPriority> => {
+    const endpoint = `${this.baseUrl}/commit/${commit}/statuses`;
+    const { data } = await axios.get<{ values: BB.BuildPriorityResponse[] }>(
+      endpoint,
+      await bitbucketAuthenticator.getAuthConfig(fromMethodAndUrl('get', endpoint)),
+    );
+
+    const allBuildStatuses = data.values;
+
+    const priority =
+      allBuildStatuses.find((buildStatus) => buildStatus.name === 'landkid-priority')
+        ?.description || 'HIGH';
```

Comment:
Should the default value here be `LOW` instead of `HIGH`?

We could also validate the `description` here as well if we field as well to ensure its one of `LOW` or `HIGH` but that may be overkill.

Reply:
Yes, the default value should be `LOW`. Missed to revert the changes after testing.

Reply:
It feels an overkill at this stage. Can be added if we plan to send more meta-data.